portal/api/portal.go

```diff
@@ -33,7 +33,6 @@ import (
 	"github.com/pairmesh/pairmesh/portal/config"
 	"github.com/pairmesh/pairmesh/portal/db"
 	"github.com/pairmesh/pairmesh/portal/sso"
-	_ "github.com/pairmesh/pairmesh/portal/sso/github"
```

Comment:
We cannot remove this line because we relay on the `github.init()` function to register sso provider.

Reply:
Sure